docs/custom-transactions.asciidoc

```diff
@@ -35,6 +35,7 @@ fetch(url)
 ----
 
 
-NOTE: There can only be one active transaction at any time.
+NOTE: By default custom transactions are not managed by the agent, therefore,
+ the agent does not create spans based on automatic instrumentations (e.g. XHR instrumentation).
```

Comment:
can you link the automatic instrumentation to the `supported technologies` doc? Might give clear understanding for the user https://github.com/elastic/apm-agent-rum-js/blob/master/docs/supported-technologies.asciidoc#supported-technologies

Reply:
+1 to V's suggestion, and one other thing:

```suggestion
the agent does not create spans based on <<supported-technologies, automatic instrumentations>>, e.g. XHR instrumentation.
```